### PR TITLE
New package: conduit-0.3.0

### PR DESCRIPTION
**Pull request by TinfoilSubmarine**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

~~Haven't played with this a lot, but it builds and runs without errors, so I figured I would put this up for PR in case anyone else is interested. I'm toying with a playground for Matrix test servers, and having all of the different implementations that are in at least Beta packaged is helpful, so I'll probably make one for Construct as well if I can get it building...~~ Working just fine for several months now.

### CODE REVIEW

**Comment by steinex** [comment]
Running this on my server and it works fine.

LGTM

**Comment by TinfoilSubmarine** [comment]
I've also been testing this more intensively and it has been working with no issues for me as well.

**Comment by steinex** [comment]
0.3.0 is out :)

**Comment by TinfoilSubmarine** [comment]
> 0.3.0 is out :)

Beat me to it, haha! Wow, both dendrite and conduit releasing on same day, what a day

**Comment by steinex** [comment]
We need to raise the ulimit for openfiles. I've run into the 1024 limit today (joined to some big rooms including HQ + running a bridge). Can you add this to run?

```
[ -r ./conf ] && . ./conf
ulimit -n ${MAX_OPEN_FILES:-8192}
```

**Comment by TinfoilSubmarine** [comment]
Sure, I'll add it and force push again once I figure out how to fix that armv6l problem.

File: srcpkgs/conduit/template [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+export BINDGEN_EXTRA_CLANG_ARGS="-I${XBPS_CROSS_BASE}/usr/include"
+if [ "$XBPS_TARGET_NO_ATOMIC8" ]; then
+	makedepends+=" libatomic-devel"
+	XBPS_CROSS_RUSTFLAGS+=" -latomic"
+fi
+
+post_install() {
+	vmkdir etc/conduit
+	vcopy conduit-example.toml etc/conduit/conduit.toml
```
Comment (jcgruenhage) [inline srcpkgs/conduit/template:30]: Sample config files should be installed using `vsconf`. That would put it into `/usr/share/examples/conduit`, which would be more appropriate anyway, IMO, as it makes it less probable that someone just installs conduit and enables it with the example config without adjusting stuff like the server domain.

**Review by jcgruenhage — CHANGES_REQUESTED** [review]

File: srcpkgs/conduit/template [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Template file for 'conduit'
+pkgname=conduit
+version=0.3.0
+revision=1
+wrksrc="conduit-v${version}"
+build_style=cargo
+build_helper=rust
```
Comment (jcgruenhage) [inline srcpkgs/conduit/template:7]: why is the build_helper included here? The cargo build_style already does that.

Comment (TinfoilSubmarine) [reply srcpkgs/conduit/template:7]: Oops, that was leftover from when I was experimenting with how the `cargo` build_style and `rust` build_helper worked.

**Comment by TinfoilSubmarine** [comment]
I dropped the BINDGEN_EXTRA_CLANG_ARGS export since it's been added since this was originally opened. I think there are no other issues with this currently, so this should be ready for review.

**Pull request merged**